roles/download/defaults/main.yml

```diff
@@ -837,11 +840,17 @@ skopeo_binary_checksums:
   ppc64l3:
     v1.10.0: 0
 
+yq_checksums:
+  amd64:
+    v4.27.5: 9a54846e81720ae22814941905cd3b056ebdffb76bf09acffa30f5e90b22d615
+    v4.30.6: 2aabdd748d301fc2286ea9f73eb20477b4ce173fbf072e0102fff1fcbed05985
+
```

Comment:
Should arm64 be added to it?
And the v4.30.6 should be first, the v4.27.5 is the next.

Reply:
Thank you for your review
Does my fix fit?